data/frontier jobs.txt

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) 2017 by Michael Zahniser and Nate Graham
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+mission "Instruct novice pilot"
+	job
+	repeat
+	description "A novice pilot seeks instruction from an experienced spacer, and requests an escort to <stopovers>. After that, return to <origin> and you will receive payment of <payment>."
+	to offer
+		random < 5
+		"combat rating" > 50
+	source
+		government Republic "Free Worlds" Neutral
```

Comment:
Per https://github.com/endless-sky/endless-sky/pull/3076#issuecomment-335306978, `Republic` -> `"Republic"`, etc.

No love for the Syndicate / Independent here?

Reply:
No Syndicate was on purpose, but no Independent was an oversight. The idea behind this is that in rougher, less economically interconnected parts of the galaxy, people are more likely to need self-reliance skills, which in this case involves learning how to pilot a starship.